lib/knapsack_pro/formatters/time_tracker.rb

```diff
@@ -0,0 +1,162 @@
+# frozen_string_literal: true
+
+module KnapsackPro
+  module Formatters
+    class TimeTracker
+      ::RSpec::Core::Formatters.register self,
+        :example_group_started,
+        :example_started,
+        :example_finished,
+        :example_group_finished,
+        :stop
+
+      # Called at the beginning of each subset,
```

Comment:
What is the subset word referring to? Is it somehow related to BuildSubset which would mean all tests assigned to a CI node?
Or is it a batch of tests fetched from queue API?
Or maybe subset is some internal RSpec wording?

Reply:
You are right, I replaced all the uses in this PR of `subset` with `batch`.